bootstrap/httpserver/server.go

```diff
@@ -47,8 +47,14 @@ func (ln tcpKeepAliveListener) Accept() (net.Conn, error) {
 	if err != nil {
 		return nil, err
 	}
-	tc.SetKeepAlive(true)
-	tc.SetKeepAlivePeriod(3 * time.Minute)
+	err = tc.SetKeepAlive(true)
+	if err != nil {
```

Comment:
all of these patterns can be better like this:
```suggestion
	if err := tc.SetKeepAlive(true); err != nil {
```